packages/netlify-cms-core/src/components/Workflow/WorkflowCard.js

```diff
@@ -84,6 +84,7 @@ const PublishButton = styled.button`
   &[disabled] {
     background-color: ${colorsRaw.grayLight};
     color: ${colorsRaw.gray};
+    cursor: not-allowed;
```

Comment:
Thanks @stefanprobst. what do you think about using the same style as https://github.com/netlify/netlify-cms/pull/5595/files#diff-ddef7e605ca14896f85ab4c0c63bd7753a2bc5e5d929429a81bd0bc9ffcd9322R244

Reply:
i honestly don't know. what motivated this change was user feedback that the (disabled) publish button "doesn't seem to be doing anything. I assume it should!" -- a "not allowed" cursor seemed like a stronger indication to me that this button is non-interactive than a "default" cursor, but it might be fine as well. i'll change.